model/gen-ai/registry.yaml

```
@@ -303,13 +303,19 @@ groups:
         note: |
           The value of this attribute matches source system tool definition format.
 
-          It's expected to be an array of objects where each object represents a tool definition. In case a serialized string is available
-          to the instrumentation, the instrumentation SHOULD do the best effort to
-          deserialize it to an array. When recorded on spans, it MAY be recorded as a JSON string if structured format is not supported and SHOULD be recorded in structured form otherwise.
+          It's expected to be an array of objects, each representing a tool definition,
+          and the structure of the array is expected to match the [Tool Definitions JSON Schema](/docs/gen-ai/gen-ai-tool-definitions.json).
+          In case a serialized string is available to the instrumentation, the instrumentation
+          SHOULD do the best effort to deserialize it to an array.
```

```suggestion
Instrumentations MUST follow [Tool Definitions JSON Schema](/docs/gen-ai/gen-ai-tool-definitions.json)
```

There is no need to mention array - it's already in the definition. Also no need to mention deserialization - it's already explicitly required by a MUST.

This is also consistent with how we define input and output messages.